frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -332,20 +332,17 @@ function FileManagerMenu:setUpdateItemTable()
         })
     end
     if not Device.should_restrict_JIT then
+        local bb = require("ffi/blitbuffer")
         table.insert(self.menu_items.developer_options.sub_item_table, {
             text = _("Disable C blitter"),
             enabled_func = function()
-                return lfs.attributes("libs/libblitbuffer.so", "mode") == "file"
+                return bb.has_cblitbuffer
             end,
             checked_func = function()
                 return G_reader_settings:isTrue("dev_no_c_blitter")
             end,
             callback = function()
-                G_reader_settings:flipNilOrFalse("dev_no_c_blitter")
-                local InfoMessage = require("ui/widget/infomessage")
-                UIManager:show(InfoMessage:new{
-                    text = _("This will take effect on next restart."),
-                })
+                bb:enableCBB(G_reader_settings:flipNilOrFalse("dev_no_c_blitter"):nilOrFalse("dev_no_c_blitter"))
```

Comment:
For readability, can you make this into 2 lines?
```lua
G_reader_settings:flipNilOrFalse("dev_no_c_blitter")
bb:enableCBB(G_reader_settings:nilOrFalse("dev_no_c_blitter"))
```

Reply:
On one hand, it's nice of luasettings to speak english predicates. On another, it returning self is utterly useless because it just encourages horrible things like this. Ideally one should be doing just 
```
bb:enableCBB(G_reader_settings:flipNilOrFalse("dev_no_c_blitter"))
```
Let's everyone now make pinkie pie promise to never use self chaining in settings predicates ever again, so this can be made lil more sensible in the future by making settings return new value whenever it flips something.